ccf_transaction_processor/scripts/configure_ccf_network.py

```diff
@@ -106,7 +107,12 @@ def Main() :
     member_cert = os.path.join(CCF_Keys, "{}_cert.pem".format(options.member_name))
     member_key = os.path.join(CCF_Keys, "{}_privk.pem".format(options.member_name))
     network_cert = config["start"]["network-cert-file"]
-    (host, port) = config["rpc-address"].split(':')
+
+    if os.environ.get("PDO_LEDGER_URL") is not None:
+        url =  os.environ.get("PDO_LEDGER_URL")
+        (host, port) = urlparse(url).netloc.split(':')
+    else :
+        (host, port) = config["rpc-address"].split(':')
```

Comment:
This if-block is repeated in a couple of scripts. I would turn this into function in a utils.py script for better reuse.

Reply:
thank you, I am adding a utils.py  within the scripts folder to avoid repetition. Please note that we cannot add the "parse" method to pdo.common.utility and use that without substantial changes to ccf installation procedure. CCF installation currently does not need PDO client to be installed.